src/game_monitor.py

```diff
@@ -13,41 +13,36 @@ class GameMonitor:
     process = None
 
     def __init__(self) -> None:
-        if not GameMonitor.process:
-            GameMonitor.get_process()
+        if not self.process:
```

Comment:
As a side note, looks like staging "selected ranges" through VSCode messed up all the code 😟
I will be commiting the rest of the code on this same branch to fix that.